### PR TITLE
Adds basic pie chart to page.

### PR DESCRIPTION
**Pull request by briafassler9**
Screenshot 2020-07-09 at 5 11 03 PM

### CODE REVIEW

**Review by Jaxobc — APPROVED** [review]

File: portfolio/src/main/webapp/statistics.html [hunk 1]
```diff
@@ -172,13 +173,18 @@ <h4>Leave a marker on the map for protests you will be attending!</h4>
         </form>
       </div>
       <div id="clear"></div>
+      <br />
+      <br />
```
Comment (kennzoid) [inline portfolio/src/main/webapp/statistics.html:177]: Prefer to use CSS to create space (unless this was actually intended to create a line break, then nvm). It's more flexible and it'll automatically apply to all elements with the same class.

File: portfolio/src/main/webapp/js/custom-script.js [hunk 2]
```diff
@@ -141,3 +140,28 @@ async function fetchSubmittedLocations() {
 
   return commentData;
 }
+
+google.charts.load('current', {'packages': ['corechart']});
+google.charts.setOnLoadCallback(drawStateIncarcerationChart);
+
+/** Creates chart and adds it to the page. */
+function drawStateIncarcerationChart() {
```
Comment (kennzoid) [inline portfolio/src/main/webapp/js/custom-script.js:148]: Non-coding comment: Mentioning the source of the data somewhere is important for credibility.

**Review by kennzoid — APPROVED** [review]

File: portfolio/src/main/webapp/statistics.html (hunk 1 above)
Comment (briafassler9) [reply portfolio/src/main/webapp/statistics.html:177]: This was used for a line break to create space between the map/chart and the end of the section.

File: portfolio/src/main/webapp/js/custom-script.js (hunk 2 above)
Comment (briafassler9) [reply portfolio/src/main/webapp/js/custom-script.js:148]: We planned to have a sources section on the statistics page where we include how where we sourced our data from.

**Pull request merged**